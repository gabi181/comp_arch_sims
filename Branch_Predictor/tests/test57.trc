4 5 13 0 global_history local_tables not_using_share
0x394AFBB0 N 0x79C912A0
0x38554D6C N 0x6028510C
0x394AFBB0 N 0x79C912A0
0x394AFBB0 T 0x79C912A0
0x38554D6C N 0x6028510C
0x38554D6C T 0x6028510C
0x38554D6C T 0x6028510C
0x38554D6C T 0x6028510C
0x394AFBB0 N 0x79C912A0
0x38554D6C T 0x6028510C
0x394AFBB0 T 0x79C912A0
0x38554D6C N 0x6028510C
0x38554D6C N 0x6028510C
0x38554D6C N 0x6028510C
0x394AFBB0 T 0x79C912A0
0x394AFBB0 N 0x79C912A0
0x38554D6C T 0x6028510C
0x394AFBB0 T 0x79C912A0
0x38554D6C T 0x6028510C
0x394AFBB0 T 0x79C912A0
0x38554D6C N 0x6028510C
