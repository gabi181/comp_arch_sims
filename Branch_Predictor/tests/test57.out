0x394afbb0 N 0x394afbb4
0x38554d6c N 0x38554d70
0x394afbb0 N 0x394afbb4
0x394afbb0 N 0x394afbb4
0x38554d6c N 0x38554d70
0x38554d6c N 0x38554d70
0x38554d6c N 0x38554d70
0x38554d6c N 0x38554d70
0x394afbb0 N 0x394afbb4
0x38554d6c N 0x38554d70
0x394afbb0 N 0x394afbb4
0x38554d6c N 0x38554d70
0x38554d6c N 0x38554d70
0x38554d6c N 0x38554d70
0x394afbb0 N 0x394afbb4
0x394afbb0 N 0x394afbb4
0x38554d6c N 0x38554d70
0x394afbb0 N 0x394afbb4
0x38554d6c N 0x38554d70
0x394afbb0 N 0x394afbb4
0x38554d6c N 0x38554d70
flush_num: 11, br_num: 21, size: 437b
